tests/test_gnps_molecular_family_loader.py

```diff
@@ -11,9 +15,31 @@ def molecular_families_gnps():
     return sut.families()
 
 
-def test_has_molecular_families():
-    filename = os.path.join(DATA_DIR, "edges.pairsinfo")
+@pytest.mark.parametrize("filename", [
+    os.path.join(DATA_DIR, "edges.pairsinfo"),
+    DATA_DIR / "edges.pairsinfo"
+])
+def test_has_molecular_families(filename):
     sut = GNPSMolecularFamilyLoader(filename)
     actual = sut.families()
     assert len(actual) == 25769
     assert len(actual[0].spectra_ids) == 19
+
+
+def test_families_are_identical(spec_dict, molecular_families: list[MolecularFamily]):
```

Comment:
```suggestion
def test_families_are_identical(spec_dict, molecular_families):
```

both parameters are fixtures, no need to provide type hint